src/ArcadeSlopes/SatSolver.js

```diff
@@ -349,33 +441,36 @@ Phaser.Plugin.ArcadeSlopes.SatSolver.prototype.collide = function (i, body, tile
  */
 Phaser.Plugin.ArcadeSlopes.SatSolver.prototype.collideOnAxis = function (body, tile, axis, response) {
 	// Bail out if we don't have everything we need
-	if (!(body.enable && body.polygon && body.slopes && tile.collides && tile.slope && tile.slope.polygon)) {
+	if (!(body.enable && body.polygon && body.slopes && tile.slope && tile.slope.polygon)) {
```

Comment:
Maybe it would be okay to put back the `tile.collides` checks. Best to completely extract this condition, too, as it's repeated elsewhere.